src/main/java/org/fundacion/jala/converter/view/converter/ConvertTypeSelectAudio.java

```diff
@@ -49,6 +53,7 @@ protected ConvertTypeSelectAudio() {
 
     /**
      * Returns the selected option to conversion.
+     *
      * @return String with the selected option.
```

Comment:
The return should be: **a** String  with the selected option.

Reply:
could you check it dani?